setup.py

```diff
@@ -25,10 +14,13 @@ def get_version():
     include_package_data=True,
     zip_safe=False,
     platforms='any',
+    # Manage version number from git at dev-time or pkg_resources at run-time.
+    setup_requires=['setuptools-git-version'],
+    version_format='{tag}',
```

Comment:
Great.

Yes, the hgtools version manages this better.

You're welcome. And @joostdevries was a great help too.